frontend/alert/src/components/AutoComplete.js

```diff
@@ -170,6 +189,7 @@ const AutoComplete = () => {
     const [inputRef, setInputRef] = useState(null);
     const [value, setValue] = useState("");
     const [suggestions, setSuggestions] = useState([]);
+    const [rawSuggestions, setRawSuggestions] = useState(null);
```

Comment:
what's the difference between `rawSuggestions` and `suggestions`? Probably a clarification in a comment would be good.